### DISCUSSION TITLE
[Feature] Option to delete or share photos from Memories

### DISCUSSION THREAD

**Discussion by Rubeast** [immich-app/immich#4496] [Feature Request]
### The feature

I really love the memories feature in Immich. 
For me this is also a great way to delete unwanted photos from my library or to share a memory with someone else. 
But at the moment we don't have any of these options. A little "More" button at the top would be great! 

Or just the standard buttons that are also used when previewing a non-memories photo like this:
![image](https://github.com/immich-app/immich/assets/5304947/03c960dc-879f-4b16-bcc0-ac54d93ad6da)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by etnoy** [comment]
Maybe just a "go to item" button to navigate to the main view of that asset

**Reply by Rubeast** [reply]
Yes. That would work too for me!

**Reply by victor-302** [reply]
This is a great feature request that unfortunately hasn't gotten too much attention.

**Comment by aniruddhas** [comment]
I wanted to download specific photos from memories on android,  couldn't do it. Came here to find issues and discussions around it, found quite a few requesting the same. 
I really hope, Immich Team notices these requests.

**Comment by flesser** [comment]
Yes, standard action buttons on (mobile) memories view would be great.

Came here because I am strongly missing "Delete" and "Favorite" during my morning routine of watching today's Immich memories.

The button to jump to the date in timeline is nice, but for busy days with ~100 photos not really usable since it is hard to find the concrete image again in the square gallery preview. It also breaks the flow.

**Reply by dpantel** [reply]
Agree with @flesser.

The “go to” button is more of a temporary bandaid solution for this problem. Please reconsider re-opening this request and implementing actual item actions from Memories.
